apps/platform-app/src/app/[locale]/(profile)/_models/profile.model.ts

```diff
@@ -16,6 +16,7 @@ export interface ProfileModel extends Profile {
   avatarUrl: string | null
   techStack: TechStack
   githubUsername: string | null
+  linkedinUsername: string | null
```

Comment:
**nit:** I know it's trivial, but let's try to stick with camelCase, co `linkedInUsername`